### PR TITLE
[Flowbroker] Remove travis and make changes to use github actions

### PR DESCRIPTION
**Pull request by marianoleonardo**
dojot/dojot#1913

### CODE REVIEW

**Review by thiagodpf — DISMISSED** [review]

**Comment by mprevide** [comment]
Did you test when the dread (docker-compose) has some error?  
Because I think It's important to see if some error happen in the dread the github actions return a status fail.

**Comment by mprevide** [comment]
> Did you test when the dread (docker-compose) has some error?
> Because I think It's important to see if some error happen in the dread the github actions return a status fail.

you might be able to change something in the code to return something different, such as a name in a key in response to a request in the API

**Comment by mprevide** [comment]
> > Did you test when the dread (docker-compose) has some error?
> > Because I think It's important to see if some error happen in the dread the github actions return a status fail.
> 
> you might be able to change something in the code to return something different/error, such as a name in a key in response to a request in the API

Did you test?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        export OWNER=$(echo ${{ github.repository }} | cut -f1 -d"/")
+        echo Current Owner is $OWNER
+        echo ::set-output name=owner::$OWNER
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Build and Push
+      uses: elgohr/Publish-Docker-Github-Action@2.16
+      with:
+        name: ${{ steps.get_owner.outputs.owner }}/flowbroker
```
Comment (mprevide) [inline .github/workflows/main.yml:76]: It's necessary to build the image `dojot/flowbroker-context-manager` too
Always look at /travis/publish.sh to see what it did.

**Review by mprevide — CHANGES_REQUESTED** [review]
I left some comments

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        node-version: ${{ matrix.node-version }}
+
+    - name: npm ci, build and check licenses
+      run: |
+        cd lib
+        npm ci
+        npm run build --if-present
+        npm run check-licenses
+
+  docker-compose:
```
Comment (mprevide) [inline .github/workflows/main.yml:35]: Integration tests?

**Review by mprevide — APPROVED** [review]

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: npm ci, build and check licenses
+      run: |
+        cd lib
+        npm ci
+        npm run build --if-present
```
Comment (mprevide) [inline .github/workflows/main.yml:32]: There're 3 packages
 - inside lib
 - inside orchestrator
 - inside contextManager

**Review by mprevide — CHANGES_REQUESTED** [review]
I left one comment

**Review by mprevide — APPROVED** [review]

**Review by thiagodpf — APPROVED** [review]

**Pull request merged**